### ISSUE TITLE
[jvm-packages]Need help! How do I get feature importance and loss function value at each iterations from SPARK XGboost?

### ISSUE THREAD

**Issue by worldtime** [dmlc/xgboost#3122]
I am running Spark Xgboost, two questions that I do not find answers from the API doc 

1. is there any function I can call to get feature importance such as Python or R version.

2, How do I get loss function value at each iteration such as 

0] train-mlogloss:1.609325+0.000006    test-mlogloss:1.609315+0.000009
[1]   train-mlogloss:1.601508+0.001238    test-mlogloss:1.602480+0.001071
[2]   train-mlogloss:1.594359+0.002151    test-mlogloss:1.596278+0.001812
[3]   train-mlogloss:1.587120+0.002100    test-mlogloss:1.589944+0.001546
[4]   train-mlogloss:1.580558+0.001839    test-mlogloss:1.584062+0.001251

.....

I really appreciate your help.

**Comment by tqchen** [comment]
General discussions are moved to https://discuss.xgboost.ai/
